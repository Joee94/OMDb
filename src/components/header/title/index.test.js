import Title from './index';

describe('title', () => {
	let returnValue;
	beforeEach(() => {
		returnValue = Title();
	});
	it('should create the header component', () => {
		expect(returnValue).toMatchSnapshot();
	});
});
