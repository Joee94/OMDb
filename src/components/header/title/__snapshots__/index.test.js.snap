// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`title should create the header component 1`] = `
<h1
  className="title"
>
  <span
    className="OMDb"
  >
    OMDb
  </span>
  <span
    className="Search"
  >
    Search
  </span>
</h1>
`;
